SmartDeviceLink/SDLOnPermissionsChange.h

```diff
@@ -22,6 +22,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (strong, nonatomic) NSArray<SDLPermissionItem *> *permissionItem;
 
+/**
+ Describes whether or not the app needs the encryption permission
+ 
+ Optional boolean available since core 5.1
```

Comment:
This should reference the RPC spec, not Core. Also I don't think 5.1 is correct.